tests/io/test_event_map.cpp

```diff
@@ -0,0 +1,34 @@
+/** TRACCC library, part of the ACTS project (R&D line)
+ *
+ * (c) 2022 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+// Project include(s).
+#include "traccc/io/event_map2.hpp"
+
+// GTest include(s).
+#include <gtest/gtest.h>
+
+// Test generate_particle_map function
+TEST(event_map2, event_map2) {
+
+    const std::string path =
+        "detray_simulation/telescope/kf_validation/1.000000_GeV_0.000000_phi/";
```

Comment:
I'm so very confused... Where should this directory be coming from? :confused:The CI seems to have found it. :confused:

Reply:
I sneakily added detray simulation data in traccc-data repository